src/Runner/CliTester.php

```diff
@@ -54,6 +54,11 @@ public function run()
 			return;
 		}
 
+		if (isset($this->options['--bootstrap'])) {
+			$exportedPath = var_export($this->options['--bootstrap'], TRUE);
+			$this->interpreter->addPhpIniOption('auto_prepend_file', $exportedPath);
```

Comment:
This should work also with HHVM [test](https://github.com/facebook/hhvm/blob/46a9a688340dd4f38806c4ca1e5798c13fb5264c/hphp/test/slow/program_functions/auto_prepend_file-1.php.ini)